utils/storage-usage-delta-calculator/src/main.rs

```diff
@@ -0,0 +1,31 @@
+use near_chain_configs::Genesis;
+use near_primitives::runtime::config::RuntimeConfig;
+use near_primitives::state_record::StateRecord;
+use node_runtime::Runtime;
+use std::fs::OpenOptions;
+use std::io::prelude::*;
+use std::io::Error;
+
+fn main() -> Result<(), Error> {
+    println!("Start");
+
+    let genesis = Genesis::from_file("output.json");
+    let mut writer = OpenOptions::new().append(true).create(true).open("delta.csv").unwrap();
+
+    println!("Genesis read");
+    let storage_usage =
+        Runtime::compute_storage_usage(&genesis.records.0[..], &RuntimeConfig::default());
+    println!("Storage usage calculated");
+    for record in genesis.records.0 {
```

Comment:
I believe this should work:

```suggestion
    let storage_usage =
        Runtime::compute_storage_usage(genesis.records.as_ref(), &RuntimeConfig::default());
    println!("Storage usage calculated");
    for record in genesis.records.into() {
```

P.S. `GenesisRecords` needs one more derive `DeriveMore::Into` to support `.into()`